src/Service.Tests/SqlTests/GraphQLMutationTests/DwSqlGraphQLMutationTests.cs

```diff
@@ -0,0 +1,604 @@
+                mutation($id: Int!, $col2Value: String) {
+                    createGQLmappings(item: { column1: $id, column2: $col2Value }) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true, new() { { "id", 2 }, { "col2Value", "My New Value" } });
+            Assert.IsTrue(actual.ToString().Contains("success"), "DwSql insert mutation with variables and mappings should be successful");
+        }
+
+        /// <summary>
+        /// Demonstrates that using mapped column names for fields within the GraphQL mutation results in successful engine processing
+        /// of the column2 value update for the record where column1 = $id.
+        /// </summary>
+        [TestMethod]
+        public async Task UpdateMutationWithVariablesAndMappings()
+        {
+            string graphQLMutationName = "updateGQLmappings";
+            string graphQLMutation = @"
+                mutation($id: Int!, $col2Value: String) {
+                    updateGQLmappings(column1: $id, item: { column2: $col2Value }) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true, new() { { "id", 3 }, { "col2Value", "Updated Value of Mapped Column" } });
+            Assert.IsTrue(actual.ToString().Contains("success"), "DwSql insert mutation with variables and mappings should be successful");
+        }
+
+        /// <summary>
+        /// <code>Do: </code>Update book in database and return the typename of the entity
+        /// <code>Check: </code>if the mutation executed successfully and returned the correct typename
+        /// </summary>
+        [TestMethod]
+        public override async Task UpdateMutationWithOnlyTypenameInSelectionSet()
+        {
+            string graphQLMutationName = "updatebook";
+            string graphQLMutation = @"
+                mutation {
+                    updatebook(id: 1, item: { title: ""Even Better Title"", publisher_id: 2345} ) {
+                        __typename
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string expected = @"
+              {
+                ""__typename"": ""DbOperationResult""
+              }
+            ";
+
+            SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
+        }
+
+        /// <summary>
+        /// Demonstrates that using mapped column names for fields within the GraphQL mutation results in successful engine processing
+        /// of removal of the record where column1 = $id and the returned object representing the deleting record utilizes the mapped column values.
```

Comment:
comment needs updating to reflect behavior of dwsql

Reply:
fixed.